.github/actions/chocolatey/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+function choco {
+  mono /opt/chocolatey/choco.exe "$@" --allow-unofficial --nocolor
```

Comment:
Instead of running this in mono in a docker container, could this just run on a windows runner instead? A similar PR for winget had the idea of just running this publish step post-publish by using
```yaml
on:
  release:
    types: [published]
```

Then we could just use `runs-on: windows-latest`, install chocolatey and use it the normal way.

Reply:
Probably, I've initially reused work from other project I maintain where it just runs as GitHub action as container